packages/console/src/components/Pagination/Next.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Next = () => {
+  return (
+    <svg width="10" height="16" viewBox="0 0 10 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Probably exported the wrong element from Figma? Icon size should be 24x24.
And also in my PR #432, I have the `ArrowUp` and `ArrowDown` in the same component class `Arrow.tsx`. So I think these ArrowLeft and ArrowRight should be put together with those two.

Reply:
Updated with new icon, size is 20x20. But the icons in your PR are 24x24, so I think we cannot put them together unless we dicide to align our icon usage.